manticore/core/cpu/arm.py

```diff
@@ -907,14 +914,20 @@ def _SR(cpu, insn_id, dest, op, *rest):
         if insn_id == ARM_INS_ASR:
```

Comment:
can we update the docstring to say that what's there only applies to arm, not thumb?